src/debugger/stepper.jl

```diff
@@ -76,8 +77,79 @@ function add_breakpoint(bp)
   JuliaInterpreter.breakpoint(bp["file"], bp["line"], cond)
 end
 
+function insert_bps!(expr)
+  i = length(expr.args)
+  for arg in reverse(expr.args)
+    if arg isa LineNumberNode
+      lnn = arg
+      for bp in JuliaInterpreter.breakpoints()
+        if bp isa JuliaInterpreter.BreakpointFileLocation
+          if string(lnn.file) == bp.abspath && lnn.line == bp.line
+            insert!(expr.args, i, JuliaInterpreter.BREAKPOINT_EXPR)
+            insert!(expr.args, i, lnn)
+            i -= 1
+          end
+        end
+      end
+    end
+    if arg isa Expr && !(arg.head in (:function, :struct))
+      insert_bps!(arg)
+    end
```

Comment:
Guess we need to handle assignment form function definition as well ? Maybe by `MacroTools.isdef(arg)`.

Reply:
Yeah, I forgot that short form function definitions don't lower the same...

Reply:
Irrelevant now since we use the upstream implementation.